README.md

```diff
@@ -55,15 +111,15 @@ To do so, first export the agent configuration.
 ```sh
 # obtained during agent onboarding
 export SUPERBLOCKS_AGENT_KEY='<agent-key>'
-export SUPERBLOCKS_AGENT_HOST_URL='http[s]://<agent-host[:port]>/agent'
-export SUPERBLOCKS_AGENT_ENVIRONMENT='<"*"|"staging"|"production">'
+export SUPERBLOCKS_AGENT_HOST_URL='http[s]://<agent-host[:port]>'
```

Comment:
Do we still need to call out the port? Or does the syntax mean it's optional?

Reply:
Yeah, the syntax makes it optional. It's only needed if it's not on the default http/https ports